src/kontrol/cli.py

```diff
@@ -168,6 +168,13 @@ def foundry_args(self) -> ArgumentParser:
             type=dir_path,
             help='Path to Foundry project root directory.',
         )
+        args.add_argument(
+            '--enum-constraints',
+            dest='enum_constraints',
+            default=None,
+            action='store_true',
+            help='Skip adding constraints for enum function arguments and storage slots.',
```

Comment:
Should this change too?
```suggestion
            help='Add constraints for enum function arguments and storage slots.',
```

Reply:
Fixed.